tests/dag_block_test.cpp

```diff
@@ -173,7 +173,7 @@ TEST_F(DagBlockTest, sign_verify) {
 
 TEST_F(DagBlockTest, push_and_pop) {
   auto node_cfgs = make_node_cfgs(1);
-  FullNode::Handle node(node_cfgs[0]);
+  auto node = create_nodes(1).front();
```

Comment:
I dont know, you are creating node_cfgs which are used later in test. Seemed much more intuitive to see the same config being used in test and also in FullNode creation compared to this where you create configs for tests and then inside create_nodes create again new configs that should and must be the same. Seems like a nice opportunity for future bugs/bad design.

Reply:
Totally agree. Will fix that